ntbs-service/Services/NotificationService.cs

```diff
@@ -434,7 +434,7 @@ public async Task UpdateMBovisDetailsExposureToKnownCasesAsync(Notification noti
         {
             _context.SetValues(notification.MBovisDetails, new
             {
-                HasExposureToKnownCases = mBovisDetails.ExposureToKnownCasesStatus
+                mBovisDetails.ExposureToKnownCasesStatus
```

Comment:
This is a pleasant use of a language feature to achieve something slightly safer (even if it isn't quite full type safety)